apps/passport-server/migrations/27_pretix_checkin_sync.sql

```diff
@@ -0,0 +1,5 @@
+alter table devconnect_pretix_tickets
+add column pretix_checkin_timestamp timestamp;
+
+alter table devconnect_pretix_events_info
+add column checkin_list_id VARCHAR not null default '0';
```

Comment:
I don't love this default. In practice it gets overwritten on first sync, and it's necessary to give the migration something to fill in as a default.

Reply:
we could wipe all the devconnect data from the database in this migration, then re-insert the configuration.

Reply:
Isn't there some code that depends on the DB being populated? It will get re-populated by sync, but in the meantime there would be no events in the DB.